com.unity.render-pipelines.high-definition/Documentation~/clouds-in-hdrp.md

```diff
@@ -6,3 +6,5 @@ The High Definition Render Pipeline (HDRP) includes two cloud solutions that you
 - [Volumetric Clouds](Override-Volumetric-Clouds.md)
 
 You can use both solutions together. In this case, you can use Volumetric Clouds for closer, interactable (receives fog and volumetric light) clouds, and Cloud Layer for distant clouds that are part of the skybox.
+
+__Note:__ Neither Cloud Layer or Volumetric Clouds support Planet Center Position.
```

Comment:
Neither should be paired with "nor" but seeing as that's kind of archaic language (and "either" doesn't work here) I recommend the following: 

__Note:__ Cloud Layer and Volumetric Clouds do not support Planet Center Position.